assets/javascripts/vector-drag.js

```diff
@@ -0,0 +1,72 @@
+var ol = require('openlayers')
+
+function VectorDrag () {
+  ol.interaction.Pointer.call(this, {
+    handleDownEvent: VectorDrag.prototype.handleDownEvent,
+    handleDragEvent: VectorDrag.prototype.handleDragEvent,
+    handleMoveEvent: VectorDrag.prototype.handleMoveEvent,
+    handleUpEvent: VectorDrag.prototype.handleUpEvent
+  })
+
+  this.coordinate_ = null
```

Comment:
I assume the underscore is to denote "private" vars? If so, I'd go with putting them at the start e.g. this._coordinate